user_guide/data_management.rst

```diff
@@ -0,0 +1,234 @@
+can adjust report to show only the data for your VIPs, and make targeted mailings to them. Moreover, you can find a 
+record by its tag with the :ref:`search <user-guide-getting-started-search>` functionality.
+More information about tags is available in the :ref:`Tags <user-guide-ui-components-view-pages>` guide.
+ 
+
+Define the Process Within Which Record Details Can Be Changed And Follow It
+---------------------------------------------------------------------------
+
+In the process of the work with the customers is ongoing: carts turn into orders, 
+what seemed to be a potential opportunity appears not to be worth effort or tuns into a successful deal, and so on. 
+These and other processes are reflected in OroCRM with a change of record properties and sometimes creation of new 
+relations. (For example - a new lead appeared, and then it turned successful and was bound to some contact, some 
+opportunity and some customer).
+
+In many cases success of the business depends a lot on the unity of the procedures followed all through the company.
+For example, most companies won't allow their employees to close a cart without contacting the potential customer but
+no one wants to annoy the customers with duplicating calls. Customer complaints have to be researched and reacted to.
+Potential big contract requires some negotiations and only after that can turn into a successful deal or a lost chance.
+
+In order to regulate this, workflows can be created in OroCRM. Workflow defines what transitions (changes of 
+properties) are available to the system users at each step.
+
+You can find more details about workflows in the :ref:`Workflows <user-guide-workflow-management-basics>` guide.
+
+Log Actions in the System
+-------------------------
+
+Whether a workflow has been defined for entity records, or users can process it in any order, it is important to know 
+what actions have been performed with a record and by whom. Who has turned this cart into an order? Who has closed 
+an opportunity? Who has deleted a customer? Who has changed contact details? In order to know the answers, you 
+can use OroCRM's data audit functionality - with it you will see what actions have been performed in the system, when 
+and by whom.
+
+Detailed description of the functionality is available in the :ref:`Data Audit <user-guide-data-audit>` guide.
+
+Integrate with Third-Party Systems
+----------------------------------
+
+OroCRM can be integrated with third-party systems, as a result of integration new records can be uploaded to OroCRM and 
+passed to the integrate system from OroCRM.
+For example, ldap integration enables loading user records, integration with Magento loads customer records from 
+a Magento store to OroCRM and integration with MailChimp will load OroCRM contact details to MailChimp and get results 
+of an email campaign back to OroCRM.
```

Comment:
Edit: For example, integrating with ldap lets you load user records, integrating with Magento lets you load customer records from a Magento store into OroCRM, and integrating with MailChimp lets you load OroCRM contact details into MailChimp, as well as get the results of an email campaign within OroCRM.